frontend/assets/javascripts/src/modules/images.js

```diff
@@ -1,16 +1,20 @@
-define(['lib/bower-components/imager.js/Imager'], function(Imager) {
+define(['lib/bower-components/imager.js/Imager', 'src/utils/helper'], function(Imager, utilsHelper) {
 
-    var IMAGES_EVENTS = '.delayed-image-load';
+    var IMAGE_LOADED_IMAGE = '.imager-loaded-image';
     var IMAGES_SLIDESHOW = '.js-image-slideshow';
 
     function init() {
 
-        if (document.querySelectorAll(IMAGES_EVENTS).length) {
-            new Imager(IMAGES_EVENTS, {
-                availableWidths: guardian.membership.eventImages.widths,
-                availablePixelRatios: guardian.membership.eventImages.ratios,
-                lazyload: true,
-                lazyloadOffset: 100
+        var imagerImages = utilsHelper.toArray(document.querySelectorAll(IMAGE_LOADED_IMAGE));
+
+        if (imagerImages.length) {
+            imagerImages.map(function(image) {
+                var imageAvailableWidths = image.getAttribute('data-available-widths');
+                new Imager([image], {
+                    availableWidths: imageAvailableWidths ? imageAvailableWidths.split(',') : [],
+                    lazyload: true,
```

Comment:
Can we put this into a data-attribute, so we can pass a lazyload boolean through to the event image fragment? Then we can opt-in to lazyloading?